cli/popper/commands/cmd_run.py

```diff
@@ -7,7 +7,7 @@
 import popper.cli
 from popper.cli import pass_context, log
 from popper.gha import WorkflowRunner
-from popper.parser import Workflow
+from popper.parser import HCLWorkflow, YMLWorkflow
```

Comment:
as a follow up to the other comment: if we have a factory method, we allow callers not to worry about the format, so we can do:

```python
from popper.parser import Workflow

# ...

Workflow.new(kwargs['wfile'])
```